libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/ComponentInstaller.kt

```diff
@@ -32,6 +31,7 @@ fun ComponentInstaller.speedLimit(
 /**
  * Speed limit view component configuration class.
  */
+@ExperimentalPreviewMapboxNavigationAPI
```

Comment:
The PR is about removing the annotation, should ie be added here?

Reply:
Yes. Refers
> not removing @ExperimentalPreviewMapboxNavigationAPI from any files related to ComponentInstaller and it's usage. ComponentInstaller could be subject to modifications and we need to play more with them before declaring them stable.